distribution/ltp-upstream/include/ltp_log_parser.py

```diff
@@ -153,15 +157,41 @@ def parse(file_handle):
     return l_tcs
 
 
+def get_known_issues(kifile):
+    known_issues = set()
+    with open(kifile, 'r') as file_handle:
+        lines = file_handle.readlines()
+        for line in lines:
+            case = line.strip().rstrip()
+            if case.startswith('#'):
+                continue
+            known_issues.add(case)
+    return known_issues
+
+
+def is_known_issue(known_issues, case):
+    for issue in known_issues:
+        if re.match(issue, case):
+            if VERBOSE:
+                print("CASE %s matched PATTERN %s" % (case, issue))
+            return True
+    return False
+
+
 def usage(prog):
-    print("Usage: {} [-r] [-t indent] [-F] <logfile>".format(prog),
+    print("Usage: {} [-r] [-t indent]".format(prog) +
+          " [-v] [-f <known-issues-file>] [-F] <logfile>",
           file=sys.stderr)
     print("e.g.",
           file=sys.stderr)
     print("o dump failed test cases",
           file=sys.stderr)
     print("       {} -F /tmp/foo.log".format(prog),
           file=sys.stderr)
+    print("       {} -f /tmp/KNOWNISSUES -F /tmp/foo.log".format(prog),
+          file=sys.stderr)
+    print("       {} -v -f /tmp/KNOWNISSUES -F /tmp/foo.log".format(prog),
```

Comment:
Did you mean to include this line twice, once with -f and again with -v -f ?

Reply:
`-v` means `verbose`. I updated the script usage to make it clear. Please refer to:
* https://gitlab.cee.redhat.com/huanli/pluto/blob/sandbox-FASTMOVING-1645/PR473/follow-up/00.md

Reply:
Great thanks for adding that!